src/getHumanAge.test.js

```diff
@@ -8,6 +8,30 @@ describe(`Function 'getHumanAge':`, () => {
   });
 
   it(`should return an array`, () => {
+    expect(getHumanAge([])).toBeInstanceOf(Array);
+  });
+
+  it(`should return 15 human years for first cat & dog year`, () => {
+    expect(getHumanAge(14, 23)).toEqual([0, 1]);
+  });
+
+  it(`should return 15 human years for first cat & dog year`, () => {
+    expect(getHumanAge(15, 14)).toEqual([1, 0]);
+  });
+
+  it(`should return 15 human years for first cat & dog year`, () => {
+    expect(getHumanAge(23, 24)).toEqual([1, 2]);
+  });
+
+  it(`should return 15 human years for first cat & dog year`, () => {
+    expect(getHumanAge(24, 23)).toEqual([2, 1]);
+  });
+
+  it(`should return 15 human years for first cat & dog year`, () => {
+    expect(getHumanAge(28, 28)).toEqual([3, 2]);
+  });
 
+  it(`should return 15 human years for first cat & dog year`, () => {
+    expect(getHumanAge(29, 29)).toEqual([3, 3]);
   });
```

Comment:
Your input doesn`t match with the description

Reply:
You said that it has to be done with the Test design technique, or I'not quite sure followed you?) I have to do this with Examples in description only?